### PR TITLE
Navbar styling

### PR DESCRIPTION
**Pull request by lmtroper**

### CODE REVIEW

File: frontend/src/components/navigator/BallIcon.jsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import React from "react";
+
```
Comment (ivanManzalez) [inline frontend/src/components/navigator/BallIcon.jsx:2]: youre gunna have to walk me through this stuff, or send me somewhere I can read up on SVGs

File: frontend/src/components/navigator/Layout.jsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import Sidebar from "./Sidebar";
+import Navbar from "./Navbar";
+
+
+const Layout = ({ children }) => {
+  return (
+    <div style={{ display: "flex" }}>
+      <Navbar />
```
Comment (ivanManzalez) [inline frontend/src/components/navigator/Layout.jsx:9]: diff b/w Navbar and Sidebar?

**Review by ivanManzalez — APPROVED** [review]
no issues

File: frontend/src/components/navigator/Layout.jsx (hunk 2 above)
Comment (lmtroper) [reply frontend/src/components/navigator/Layout.jsx:9]: Navbar is at the top whereas sidebar is the navigation on the side

**Pull request merged**